tests/python_tests/samples/test_encrypted_model_causal_lm.py

```diff
@@ -12,17 +12,17 @@ class TestEncryptedLM:
     @pytest.mark.llm
     @pytest.mark.samples
     @pytest.mark.parametrize("convert_model", ["Qwen2.5-0.5B-Instruct"], indirect=True)
-    @pytest.mark.parametrize("sample_args", ["Why is the sun yellow?"])
+    @pytest.mark.parametrize("prompt", ["Why is the sun yellow?"])
 
-    def test_sample_encrypted_lm(self, convert_model, sample_args):
+    def test_sample_encrypted_lm(self, convert_model, prompt, tmp_path):
```

Comment:
Where tmp_path is set?

Reply:
https://docs.pytest.org/en/4.6.x/tmpdir.html
It is default fixture